### ISSUE TITLE
How to train multiclass documents data?

### ISSUE THREAD

**Issue by shashankg7** [alexander-rakhlin/CNN-for-Sentence-Classification-in-Keras#5]
Hi,

Thanks for releasing the code in keras, the other implementations are in torch and tensorflow  and theano which currently I am not familiar with. 

I have one question though, how can I train multiclass documents with this model? Currently it seems to support only binary class data?

**Comment by alexander-rakhlin** [comment]
Thanks for the interest.
I suggest softmax as last layers:

`model.add(Dense(num_classes))
model.add(Activation('softmax'))
model.compile(loss='categorical_crossentropy', optimizer='rmsprop', metrics=['accuracy'])`

**Comment by GoXian** [comment]
> 感谢您的关注。我建议在最后一层使用softmax，categorical_crossentropy作为损失：
> 
> ```
> model.add(Dense(num_classes))
> model.add(Activation("softmax"))
> model.compile(loss="categorical_crossentropy", optimizer="rmsprop", metrics=["accuracy"])
> ```

你好
我看见论文中写的是使用多个 Sigmoid 函数而不是 使用Softmax 函数。我的理解是如果使用Softmax 函数 是不能解决 OSR 问题。

**Comment by worksduke** [comment]
来信收悉。
